mapbook-iOS/View controllers/LocalPackagesListViewController.swift

```diff
@@ -197,16 +206,10 @@ class LocalPackagesListViewController: UIViewController {
             controller.mobileMapPackage = package
         }
         else if segue.identifier == "PortalURLSegue",
-            let controller = segue.destination as? PortalURLViewController {
-            
-            controller.delegate = self
-            controller.preferredContentSize = CGSize(width: 400, height: 300)
-            controller.presentationController?.delegate = self
-        }
-        else if segue.identifier == "UserProfileSegue",
-            let controller = segue.destination as? UserProfileViewController {
+            let controller = segue.destination as? PortalAccessViewController {
             
             controller.delegate = self
+            controller.preferredContentSize = CGSize(width: 400, height: 530)
```

Comment:
I _think_ this can be done in the storyboard by selecting the `Portal Access View Controller` and in the Attributes panel setting the `Content Size: User Preferred Explicit Size` checkbox. Might just be nicer to have that in the storyboard rather than hard-coding here:

![2018-10-14_13-25-08](https://user-images.githubusercontent.com/25374/46919935-cf131f80-cfb4-11e8-862f-ff34519041b3.png)

Reply:
Oy, this technique was left over from before. I've now taken your suggestion.